pkg/apis/batch/validation/validation.go

```diff
@@ -544,14 +544,6 @@ func validateTimeZone(timeZone *string, fldPath *field.Path) field.ErrorList {
 	return allErrs
 }
 
-// ValidateJobTemplate validates a JobTemplate and returns an ErrorList with any errors.
```

Comment:
Removed because I noticed we weren't covering this function in any test case.  And it wasn't used at all in the code.  

@alculquicondor Do you think this function should actually be used?

Reply:
The `batch.JobTemplate` object doesn't seem to have a counterpart in `staging/k8s.io/api/batch/v1`. I think we can get rid of that too.

@liggitt @soltysh. These are just leftovers from ScheduledJob, right? https://github.com/kubernetes/kubernetes/pull/21675

Reply:
I’m confused. I think we use JobTemplate in the Cronjob?

Reply:
So you are saying that https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/batch/v1beta1/types.go#L30 is not used?

I am a little confused at the v1beta1 and v1 scope under api/batch.  Are we using v1beta1 and v1?  I did not see anything related to JobTemplate in v1 but it is present in v1beta1.

Reply:
Again, it looks like a leftover from the old ScheduledJob (not CronJob).